src/modules/action-button/action-button.component.scss

```diff
@@ -0,0 +1,14 @@
+/deep/
+.stache-action-buttons {
+    display: inline-flex;
+    flex-wrap: wrap;
+
+  sky-action-button {
```

Comment:
Styles on classnames are (albeit negligibly) faster than element names. Just good practice to style against classnames when possible. http://stackoverflow.com/questions/39111878/which-css-selector-is-faster-in-performance-class-in-class-vs-tagname-in-class